### PR TITLE
Tickets/dm 41772

### PR DESCRIPTION
**Pull request by wvreeven**

### CODE REVIEW

File: doc/version_history.rst [hunk 1]
```diff
@@ -18,6 +18,8 @@ v2.0.0
 * Reorganize constants and event information.
 * Add support for sending start, disable, enable and standby commands.
 * Fix typo in the "ATMCS_nasmyth_m3_mountMotorEncoders" telemetry topic name.
+* Add support for summary state and detailed state events and for FAULT state.
```
Comment (teweitsai) [inline doc/version_history.rst:21]: state events and for FAULT state. --> state events for FAULT state.

Or I misunderstood?

File: python/lsst/ts/atmcssimulator/mcs_csc.py [hunk 2]
```diff
@@ -47,23 +48,27 @@ class ATMCSCsc(attcpip.AtTcpipCsc):
...
+            override=override,
+            simulation_mode=simulation_mode,
         )
 
         # McsSimulator for simulation_mode == 1.
         self.simulator: McsSimulator | None = None
 
     async def start_clients(self) -> None:
-        if self.simulator is None:
+        if self.simulation_mode == 1 and self.simulator is None:
```
Comment (teweitsai) [inline python/lsst/ts/atmcssimulator/mcs_csc.py:71]: Maybe:

```
if (self.simulation_mode == 1) and (self.simulator is None):
```

**Review by teweitsai — APPROVED** [review]
Looks good.

File: doc/version_history.rst (hunk 1 above)
Comment (wvreeven) [reply doc/version_history.rst:21]: I added support for two things:
- the events for summary state and detailed state
- the FAULT state

File: python/lsst/ts/atmcssimulator/mcs_csc.py (hunk 2 above)
Comment (wvreeven) [reply python/lsst/ts/atmcssimulator/mcs_csc.py:71]: I know that increases readability but the parenthesis are not necessary. I prefer the version without them.

**Pull request merged**